rizza/helpers/config.py

```diff
@@ -31,7 +41,7 @@ def __attrs_post_init__(self):
             self.cfg_file = Path().joinpath(self.cfg_file)
         elif self.cfg_file != str(Path(self.cfg_file).absolute()):
             self.cfg_file = self.base_dir.joinpath(self.cfg_file)
-        self.RIZZA["CONFILE"] = self.cfg_file
+        self.RIZZA["CONFILE"] = str(self.cfg_file)  # Convert to string for serialization
```

Comment:
Since NanoConf uses Python-Box, we can switch to attribute access for nested values. Please do that switch throughout.
```suggestion
        self.RIZZA.CONFILE = str(self.cfg_file)  # Convert to string for serialization
```